### ISSUE TITLE
Opening summary to include "thermometer"

### ISSUE THREAD

**Issue by terracoda** [phetsims/friction#104]
A side effect from making shorter alerts was that I removed the word "thermometer" from the alerts.

I think it is important that users know there is indeed a thermometer present, so I added this word to the opening summary paragraph.

I don't think I communicated the changes to the second sentence very clearly. There are effectively 2 levels of change. There are three non-transitioning summaries that include the word "thermometer", and do not describe a changing state, but rather a settled state. In these settled scenarios, the jiggle is described first and then the temperature.

And then, there are three transitioning scenarios that describe a changing state, and these scenarios drop the word "thermometer" and swap the order of the description temperature (i.e. dropping temperature) and then the slowing jiggle of the atoms.

For example, on page load, the opening paragraph should be:
"Chemistry book rests on top of a Physics book, and is ready to be rubbed against it. In zoomed-in view of where books meet, _atoms jiggle a tiny bit, and surface temperature **thermometer** is cool_. Move Chemistry book to rub books together."

Currently, I hear:
>Chemistry book rests on top of a Physics book, and is ready to be rubbed against it. In  zoomed-in view of where books meet, _**atoms** surface temperature is cool and atoms jiggle **jiggle** a tiny_. Move Chemistry book to rub books together."

I mentioned the double jiggle over in issue #93 before I realized that the issue was a little more complex.

So logic-wise if things are settled it will be scenario 1, 2, or 3 as described in the design document, and if things are in transition, we have the same three scenarios, but the second sentence is in transition rather than settled.

@zepumph, does that make sense?
section in design document: [Scenarios for Describing the State of Friction](https://docs.google.com/document/d/1v7GM_dNZs86jcRn2X03D2t7HFDb4dQ1h4ayDGXTu63U/edit#heading=h.3kz4xdtel5zt)

**Comment by terracoda** [comment]
@zepumph, you might want to hold of on this issue until I work on the last sentence of the summary. 

Based on our Sept 5th meeting I am trying to come up with something that communicate something about the books being pressed together or not pressed together without directly saying that ;-)

**Comment by terracoda** [comment]
@zepumph, I completed some rewording of the scene summary  to address #111 which will discuss at our Friction meeting today. 

I will close this issue as it will be fixed once we decide on our wording changes for #111.

**Comment by terracoda** [comment]
@zepumph, just re-opening this issue so it doesn't get lost. The summary will need to include all changes in the design document (see also #111).

**Comment by zepumph** [comment]
I will cover this in https://github.com/phetsims/friction/issues/111
